repostatus.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
+from dotenv import load_dotenv
+import json
+
+# Load environment variables
+load_dotenv(r'C:\Users\home\tf2.4\TF2.4_IVIM-MRI_CodeCollection\local.env')
+
+# Read the CSV file
+csv_file_path = os.getenv('CODE_CONTRIBUTIONS_FILE')
```

Comment:
Looking through the code I realize that your table is missing a column, `wrapped`. In the [issue thread](https://github.com/OSIPI/TF2.4_IVIM-MRI_CodeCollection/issues/47#issuecomment-1973909276) I mentioned parsing the files in the folders which isn't being done here, and that's what is missing. It should parse the wrapper folder to find which algorithms are wrapped.

And there I also mentioned double checking the code contributions against what is in the source folder.

So the source code folder and code contribution file should match. If not it would be nice to note that. That should be a superset of the wrapped code list. And that should be a superset of the tested code list. If anything doesn't match that logic then we should note it in the html or printing a message.